tools.yml

```diff
@@ -1428,12 +1395,14 @@ tools:
     mem: 24
   toolshed.g2.bx.psu.edu/repos/iuc/iqtree/iqtree/.*:
     cores: 10
+    mem: 7
```

Comment:
This is low setting for mem. Max memory for Galaxy Australia's iqtree jobs is regularly in the 8-19Gb range (19 is the what we give it in TPV)